packages/x-components/src/x-modules/url/config.types.ts

```diff
@@ -3,4 +3,11 @@
  *
  * @public
  */
-export interface UrlConfig {}
+export interface UrlConfig {
+  query?: string;
+  page?: string;
+  filters?: string;
+  sort?: string;
+  relatedTags?: string;
+  [extraParam: string]: string | undefined;
```

Comment:
I'm not sure whether the behaviors I stated in the PR description (excluding certain values) will be hardcoded , or should also be part of some configuration. If they should be configurable, I would nest right now these parameters under `UrlConfig['mappings']`

Reply:
What would those mappings be? The resultant values of mapper functions that would be part of the configuration?